s4e-backend/src/main/java/pl/cyfronet/s4e/service/GroupService.java

```diff
@@ -110,4 +174,41 @@ public void update(Group group) throws GroupUpdateException {
             throw new GroupUpdateException("Cannot update Group", e);
         }
     }
+
+    @Transactional(rollbackFor = GroupUpdateException.class)
+    public void updateFromRequest(UpdateGroupRequest request, String institutionSlug, String groupSlug) throws GroupUpdateException, NotFoundException {
+        Set<String> membersEmailsFromDB = groupRepository.findAllMembersEmails(institutionSlug, groupSlug, AppRole.GROUP_MEMBER);
+        Set<String> difference = membersEmailsFromDB.stream().collect(Collectors.toSet());
```

Comment:
Why is the `difference` variable not initialized directly from the repo?
I'd call the `difference` maybe `membersToRemove`, `difference` is unclear.